Lib/importlib/_bootstrap_external.py

```diff
@@ -1163,6 +1163,9 @@ def _recalculate(self):
     def __iter__(self):
         return iter(self._recalculate())
 
+    def __getitem__(self, index):
+        return self._path[index]
```

Comment:
Should not we call `self._recalculate()`?

Reply:
hmm yeah probably -- let me fix that